src/Bicep.LangServer/Snippets/Snippet.cs

```diff
@@ -14,11 +14,11 @@ namespace Bicep.LanguageServer.Snippets
     public sealed class Snippet
     {
         // Below regex is used to detect following snippet syntax:
-        // ${(?<index>\d+):(?<name>\w+)} detects placeholders i.e. tab stops with values e.g ${1:foo}
+        // ${(?<index>\d+):(?<name>\w+|\*|(\w+\/\w+)+@(\w+|(\d{4}-\d{2}-\d{2})))} detects placeholders i.e. tab stops with values e.g ${1:foo}
```

Comment:
Could this regex be simplified to just:
```
\${(?<index>\d+):(?<name>[^}]+)}
```

Reply:
Sure, updated here: https://github.com/Azure/bicep/commit/535fbff32bc2dd3083591076309d3dffdb1c93fe